addons/website_event_exhibitor/controllers/exhibitor.py

```diff
@@ -174,6 +174,7 @@ def _event_exhibitor_get_values(self, event, sponsor, **options):
             'option_widescreen': option_widescreen,
             'option_can_edit': request.env.user.has_group('event.group_event_user'),
             # environment
+            'description_is_empty': not html2plaintext(sponsor.website_description).strip(),
```

Comment:
does `is_html_empty` work ?

Reply:
No, but actually I can just add `section` to the regex in there instead. Should be cleaner.

Also while in there, seems odd to compile the regex inside that function. Doesn't seem like we gain anything from that since we only use it in `sub`, which already compiles it if not compiled. Probably not something to change now though.